### ISSUE TITLE
Build Mode with Watch Giving Incomplete Output

### ISSUE THREAD

**Issue by blink1073** [Microsoft/TypeScript#26955]
**TypeScript Version:**  3.0.3

**Search Terms:**  build watch

**Code**
Clone TypeScriptSamples
cd simple
tsc --build . --watch
Change animals.ts

**Expected behavior:**
Change the file multiple times, getting build updates each time, e.g.:

```
[06:02:47] File change detected. Starting incremental compilation...

[06:02:48] Found 0 errors. Watching for file changes.
```

**Actual behavior:**
Build update is only shown once, thereafter you only get the `File change detected` notice.

**Related Issues:** This is a follow up to https://github.com/Microsoft/TypeScript/issues/26147 that was closed by https://github.com/Microsoft/TypeScript/pull/26590.  @mjbvz had asked for a backport to 3.0.x for use in VSCode.  The behavior of `typescript@next` is correct, but not 3.0.3.

**Comment by sheetalkamat** [comment]
@RyanCavanaugh This is fixed by #26747 which is not part of 3.0.3

**Comment by RyanCavanaugh** [comment]
Please try in 3.1 RC or `typescript@next`. Thanks!
